docs/core/linux-prerequisites.md

```diff
@@ -22,25 +22,26 @@ This article shows the dependencies needed to develop .NET Core applications on
 
 .NET Core 2.x is supported on the following Linux distributions/versions:
 
- * Red Hat Enterprise Linux 7 x64
- * CentOS 7 x64
- * Oracle Linux 7 x64
- * Fedora 25, 26 x64
- * Debian 9, 8.7+ x64 
- * Ubuntu 17.04, 16.04, 14.04  x64
- * Linux Mint 18, 17 x64
- * openSUSE 42.2+ x64
- * SUSE Enterprise Linux (SLES) 12 SP2+ x64
+ * Red Hat Enterprise Linux 7 x86_64
```

Comment:
This might be a terminology issue: the architecture that Microsoft calls `x64` is generally referred to as `x86_64` or `amd64` by most Linux distros. What Microsoft calls `x86` is generally called `x86` or `i386` or something similar.